src/inference/src/dev/core_impl.cpp

```diff
@@ -1144,7 +1148,9 @@ std::shared_ptr<const ov::Model> ov::CoreImpl::apply_auto_batching(const std::sh
         break;
     case ov::details::NetworkBatchAbility::WITH_HETERO:
         deviceName = "HETERO:BATCH," + deviceNameWithoutBatch;
+        OPENVINO_SUPPRESS_DEPRECATED_START
         config[CONFIG_KEY(AUTO_BATCH_DEVICE_CONFIG)] = batchConfig;
```

Comment:
I suppose we can use new properties here

Reply:
It seems "AUTO_BATCH_DEVICE_CONFIG" will be deprecated?

Reply:
yes, because it's an old name.
We can use universal `ov::device::priorities`